packages/router/utils.ts

```diff
@@ -248,10 +249,11 @@ export interface DataStrategyMatch
 export interface DataStrategyFunctionArgs<Context = any>
   extends DataFunctionArgs<Context> {
   matches: DataStrategyMatch[];
+  fetcherKey: string | null;
```

Comment:
We are passing the `fetcherKey` into `dataStrategy` now for fetcher cals which allows Remix to make one-off calls and not run any parent handlers in the single fetch request